libs/medic-users-meta/migrations/202105171933.do.86.usersMeta.sql

```diff
@@ -24,26 +24,29 @@ CREATE INDEX idx_useview_feedback_period_start_user ON useview_feedback(period_s
 CREATE MATERIALIZED VIEW useview_telemetry AS
 SELECT
   doc->>'_id' AS uuid,
-  CONCAT_WS(                                --> Date concatenation from JSON fields, eg. 2021-5-17
-    '-',
-    doc#>>'{metadata,year}',                --> year
-    CASE                                    --> month of the year
-      WHEN
-        string_to_array(substring(doc#>>'{metadata,versions,app}' FROM '(\d+.\d+.\d+)'),'.')::int[] < '{3,8,0}'::int[]
-      THEN
-        (doc#>>'{metadata,month}')::int+1   --> Legacy, months zero-indexed (0 - 11)
-      ELSE
-        (doc#>>'{metadata,month}')::int     --> Month is between 1 - 12
-    END,
-    CASE                                    --> day of the month, else 1
-      WHEN
-        (doc#>>'{metadata,day}') IS NOT NULL
-      THEN
-        doc#>>'{metadata,day}'
-      ELSE
-        '1'
-    END
-  )::date AS period_start,
+  (CASE 
```

Comment:
i don't think migrations run twice, so you'll have to create new which drop and replace this view instead of editing this definition

Reply:
Thanks, I've made the changes. 

I had to use `drop cascade` since materialized views can not be replaced like view `CREATE OR REPLACE`.

Reply:
that's pretty scary then... who knows what we are going to cascade delete... 
i'm not sure how we message this - but i think we need to be really clear about this somehow. can you ask on #development or something?